src/core/server/saved_objects/saved_objects_service.test.mocks.ts

```diff
@@ -30,3 +30,7 @@ export const registerRoutesMock = jest.fn();
 jest.doMock('./routes', () => ({
   registerRoutes: registerRoutesMock,
 }));
+
+// The SavedObjectsSerializer imports SavedObjectUtils from the '../service' module, and that somehow breaks unit tests for the
+// SavedObjectsService. To avoid this, we mock the entire './serialization' module, since we don't need it for these tests.
+jest.mock('./serialization');
```

Comment:
I ran into similar issues working on v2 migrations when we still had v1 around. 
In my case, the issue was with `SavedObjectsService` using the `migrator$` that depends on functionality in `serialization` and we ended up with a circular dependency there. Mocking the whole module in this case seems ok, since we're not directly using it in the tests.

Reply:
Ah, thanks!